go/mathutil/equivalence_relation_test.go

```diff
@@ -174,3 +174,119 @@ func TestEquivalenceRelation(t *testing.T) {
 		})
 	}
 }
+
+func TestUnknownElementError(t *testing.T) {
+	err := &UnknownElementError{element: "test_element"}
+
+	assert.Equal(t, "unknown element test_element", err.Error())
```

Comment:
Can you use `assert.EqualError` here and in other similar tests?

Reply:
done